### PR TITLE
UI-309 Update output for service:checks plan error

### PR DESCRIPTION
**Pull request by justinanastos**
Also fixes pluralization in the Listr status messages.

All errors thrown will cause Listr to look strange. I was unable to quickly diagnose where the error originates. It exists in prod also so I didn't try too hard to fix it.

TODO:

- [x] Update CHANGELOG.md\* with your change (include reference to issue & this PR)
- [x] Make sure all of the significant new logic is covered by tests
- [x] Rebase your changes on master so that they can be merged easily
- [x] Make sure all tests and linter rules pass

### CODE REVIEW

File: packages/apollo/src/commands/service/check.ts [hunk 1]
```diff
@@ -13,13 +13,18 @@ import {
 import { ApolloConfig } from "apollo-language-server";
 import moment from "moment";
 import sortBy from "lodash.sortby";
+import stripANSI from "strip-ansi";
 
 function pluralize(
-  quantity: number | null,
+  quantity: string | number | null,
   singular: string,
   plural: string = `${singular}s`
```
Comment (evans) [inline packages/apollo/src/commands/service/check.ts]: 😮 that's awesome!

File: packages/apollo/src/commands/service/check.ts [hunk 2]
```diff
@@ -287,10 +287,6 @@ export default class ServiceCheck extends ProjectCommand {
                 // Save the output because we're going to use it even if we throw. `runTasks` won't return
                 // anything if we throw.
                 Object.assign(taskOutput, ctx);
-
-                task.title = `Validated local schema against tag ${chalk.blue(
```
Comment (evans) [inline packages/apollo/src/commands/service/check.ts]: I think this is still valuable for the tense change

Comment (justinanastos) [reply packages/apollo/src/commands/service/check.ts]: Good call.

**Comment by justinanastos** [comment]
@evans 

> Also is there a way to test the screenshot below?

We _could_ refactor the functions that generate the messages out and then unit test them. I think we'd be better served spending a few hours to put together e2e tests. I think if we treat this like JavaScript that we can test all the output with relative ease!

**Comment by justinanastos** [comment]
@evans I updated the tense change like you requested; that's the only change.

https://github.com/apollographql/apollo-tooling/pull/1178/files#diff-c61805bdfe85787383d4ae9927bd0774R291

**Comment by justinanastos** [comment]
@evans 

>I agree with your assessment that end-2-end tests would be valuable. Do you think that it's worth tracking as a ticket in Jira?

I sure do

**Pull request merged**